app/models/entry_csv_generator.rb

```diff
@@ -27,4 +27,11 @@ def generate
       end
     end
   end
+
+  def options
+    {
+      col_sep: ";",
+      row_sep: "\r\n"
+    }.reject{ |key,value| I18n.locale == :en }
```

Comment:
Unused block argument - `key`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `value`. You can omit all the arguments if you don't care about them.<br>Space missing after comma.<br>Space missing to the left of {.